src/components/Footer/FooterBase.tsx

```diff
@@ -0,0 +1,130 @@
+import Link from '../Link';
+import logoTagline from '../../stories/assets/logo-bsc-tagline.svg';
+import { RequireOnlyOne, NavItem } from '../../types';
+import Facebook from '../../stories/assets/icon-facebook.svg';
+import Twitter from '../../stories/assets/icon-twitter.svg';
+import LinkedIn from '../../stories/assets/icon-linkedin.svg';
+import YouTube from '../../stories/assets/icon-youtube.svg';
+import NewWindowIcon from '../../../public/assets/images/icon-new-window.svg';
```

Comment:
we have a mix of naming (logoTagline, NewWindowIcon, Facebook). can we standardize with last name first conventions?